tests/CMakeLists.txt

```diff
@@ -61,6 +61,7 @@ if (UNIX OR ADD_WINDOWS_ENCLAVE_TESTS OR USE_CLANGW)
         add_subdirectory(sealKey)
         add_subdirectory(stdc)
         add_subdirectory(syscall)
+        add_subdirectory(argv)
```

Comment:
Consider building this test for TrustZone, too.

`argv/enc/enc.c` will need to use `OE_SET_ENCLAVE_OPTEE` like in `mixed_c_cpp/enc/enc.cpp`.

Reply:
THank you Hernan. We will work on this after we merge this PR.